lib/diff.js

```diff
@@ -39,7 +39,7 @@ class Diff extends BaseCommand {
   }
 
   async diff (args) {
-    const specs = this.npm.flatOptions.diff.filter(d => d)
+    const specs = this.npm.config.get('diff').filter(d => d)
```

Comment:
Not sure what this filter is doing cc @ruyadorno

Reply:
also, `.filter(Boolean)` :-)

Reply:
It's filtering out falsey values.